src/main/java/uk/gov/hmcts/reform/unspec/enums/YesOrNo.java

```diff
@@ -0,0 +1,10 @@
+package uk.gov.hmcts.reform.unspec.enums;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public enum YesOrNo {
+    @JsonProperty("Yes")
```

Comment:
could use JsonNaming annotation here on class level and UPPER_CAMEL_CASE class, not massively fussed though

Reply:
I will pass this one.